src/resolvers.js

```diff
@@ -0,0 +1,52 @@
+var util = require('util');
+var http = require('http');
+
+var baseUrl;
```

Comment:
having this variable in the top scope of the module will not allow you to call the create function multiple times with different values. Just keeping it within the function should preserve the value in the context for the inner function.